test/integration/editors/triggered/CommunicationMappingPlugin.test.ts

```diff
@@ -0,0 +1,192 @@
+import { expect, fixture, html } from '@open-wc/testing';
+
+import { MockPendingWizard } from '../../../mock-pending-wizard.js';
+import CommunicationMappingPlugin from '../../../../src/triggered/CommunicationMapping.js';
+import { List } from '@material/mwc-list';
+
+describe('CommunicationMappingPlugin', () => {
+  let doc: Document;
+  customElements.define(
+    'communication-mapping-plugin',
+    CommunicationMappingPlugin
+  );
+  let parent: MockPendingWizard;
+  let element: CommunicationMappingPlugin;
+
+  beforeEach(async () => {
+    parent = await fixture(
+      html`<mock-wizard-editor
+        ><communication-mapping-plugin></communication-mapping-plugin
+      ></mock-wizard-editor>`
+    );
+
+    element = <CommunicationMappingPlugin>(
+      parent.querySelector('communication-mapping-plugin')!
+    );
+
+    doc = await fetch('public/xml/testfiles/comm-map.scd')
+      .then(response => response.text())
+      .then(str => new DOMParser().parseFromString(str, 'application/xml'));
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7203429)
```suggestion
      .then(str => new DOMParser().parseFromString(str, "application/xml"));
```